src/index.ts

```diff
@@ -60,24 +60,20 @@ export interface Settings {
  * Check if `o` contains all the expected fields to be a [[Config]].
  */
 function validateConfig(o: JsObject) {
-    if (typeof o !== 'object') {
-        throw Error('the configuration must be a JavaScript object');
+    if (!('meta' in o && (typeof o.meta === 'string' || isHTMLElement(o.meta)))) {
+        throw Error('missing "meta" key in chemiscope config');
     }
```

Comment:
Looks like you removed this while resolving conflicts, could you put it back please?